### DISCUSSION TITLE
Modules registering for Web Component mode is not working

### DISCUSSION THREAD

**Discussion by robinsimonklein** [nolimits4web/swiper#6637] [Q&A]
Hi there,

I've been trying to get [Swiper Element](https://swiperjs.com/element) to work with only the modules I need for several hours now. I have absolutely no idea what I am doing wrong... Even if I only use what is [in the documentation](https://swiperjs.com/element#core-version-and-modules), neither pagination nor navigation works. It seems that the elements are present in the shadow-root but nothing appears and no style is applied.

Here is a minimal reproduction link based on the documentation : https://codesandbox.io/p/sandbox/jx587f

What am I doing wrong? Or is it a Swiper Element bug?

Thanks in advance for your help!

**Comment by jaheemprevost** [comment]
Were you ever able to find a solution?
